integreat_cms/cms/models/contact/contact.py

```diff
@@ -61,7 +63,19 @@ def __str__(self) -> str:
 
         :return: A readable string representation of the contact
         """
-        return f"{self.title} {self.name}"
+        point_of_contact_for = (
+            f"{self.point_of_contact_for.strip()}: "
+            if self.point_of_contact_for and self.point_of_contact_for.strip()
+            else ""
+        )
+        name = f"{self.name.strip()} " if self.name and self.name.strip() else ""
+        details = [
+            detail
+            for x in ["email", "phone_number", "website"]
+            if (detail := self.get(x)) and detail.strip()
+        ]
```

Comment:
```suggestion
        details = [
            detail
            for detail in [self.email, self.phone_number, self.website]
            if detail
        ]
```

Did you mean something like that? Because there exists no `get` method